packages/providers/email-nodemailer/.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
```

Comment:
There is an `.editorconfig` file at the root of the project, it isn't enough?

Reply:
All the other providers had it so I just mirrored the same

Reply:
Ok, I'll make a PR to remove them all I think and check if @alexandrebodin has an objection. You can let this file and I'll remove it in my PR